lr-cli/cortx_setup/commands/server/config.py

```diff
@@ -0,0 +1,84 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+from ..command import Command
+from cortx.utils.conf_store import Conf
+from provisioner.commands import PillarSet
+from cortx_setup.config import CONFSTORE_CLUSTER_FILE
+
+
+class ServerConfig(Command):
+
+    _args = {
+        'name': {
+            'type': str,
+            'default': None,
+            'optional': True,
+            'help': 'Set name of the server'
+        },
+        'type': {
+            'type': str,
+            'default': None,
+            'optional': True,
+            'choices': ['VM', 'HW'],
+            'help': 'Server type e.g {VM|HW}'
+        }
+    }
+
+
+    def run(self, **kwargs):
+        """
+        Server configuration command execution method
+
+        Command:
+        `cortx_setup server config --name <name> --type {VM|HW}}`
+        """
+
+        conf_pillar_map = {
+            'name': (
+                'server/name',
```

Comment:
should it be something like
```
pillar_key : cluster/srvnode-0/type
confstore: server_node>{machine_id}>type
```
?

Reply:
@mazinamdar @83bhp will the node_id be present at the time of server config?

Reply:
at initial every node will be srvnode-0 , once we do bootstrap we get actual node_id